src/components/vehicleDataGroup/VehicleDataGroupField.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, ReactNode } from 'react';
+import { Box, chakra, ListItem, useMultiStyleConfig } from '@chakra-ui/react';
```

Comment:
We already have the `Box` component extracted. Please use it instead using it from `@chakra-ui/react`.
Thank you!